frontend/src/v5/ui/components/shared/modalsDispatcher/templates/deleteModal/deleteModal.component.tsx

```diff
@@ -40,6 +40,7 @@ import { formatMessage } from '@/v5/services/intl';
 import { useState } from 'react';
 import { Gap } from '@controls/gap';
 import { Button } from '@controls/button';
+import { SubmitButton } from '@controls/submitButton';
```

Comment:
![image](https://github.com/3drepo/3drepo.io/assets/62481250/20968f38-294e-4703-aeda-49a065e2482b)
Not aligned